Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -50,4 +50,56 @@
   parent: WeaponRevolverDeckardExpedition
   id: WeaponRevolverDeckardExpeditionGrifty
   noSpawn: true
-  name: Customer Service Resolver
\ No newline at end of file
+  name: Customer Service Resolver
+
+- type: entity
+  parent: BaseItem
+  name: The Arbitrator
+  id: WeaponRevolverarbitrator
+  description: Based off of an older design. S.E.S.W.C decided to revamp and rechamber for a modern era  .45cal, .50cal, and .60cal. 
+  components:
+  - type: Sprite
+    scale: 0.75 , 0.75
+    sprite: _NF/Objects/Weapons/Guns/Revolvers/arbitrator.rsi
+    state: icon
+  - type: Clothing
+    sprite: _NF/Objects/Weapons/Guns/Revolvers/arbitrator.rsi
+    quickEquip: false
+    slots:
+    - Belt
+  - type: Item
+    size: Small
+    shape:
+    - 0,0,1,0
+    - 0,1,0,1
+  - type: Tag
+    tags:
+    - Sidearm
+  - type: Gun
+    minAngle: 1
+    maxAngle: 8
+    selectedMode: SemiAuto
+    fireRate: 2
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/revolver.ogg
+  - type: AmmoCounter
+  - type: RevolverAmmoProvider
+    proto: null
+    capacity: 4
+    chambers: [ null, null, null, null ]
+    ammoSlots: [ null, null, null, null ]
+    soundEject:
+      path: /Audio/Weapons/Guns/MagOut/revolver_magout.ogg
+    soundInsert:
+      path: /Audio/Weapons/Guns/MagIn/revolver_magin.ogg
+    whitelist:
+      tags:
+      - ShellShotgun
+      - CartridgeAntiMateriel
+      - CartridgeMagnum
```

Comment:
I can kinda get behind the idea of using shotgun shells and .60 interchangeably because those cartridges are similar in size, but .45 is no where near the size of either of them. I don't think .45 should be there at all.

Reply:
The original idea was to make the Judge, which is chamber in 45 and 410(a shotgun shell) and 454 casli.  Checkraze in the discord sign off on the shotgun shell. He also said 50cal. (Close we have to It was shotgun and 60anti) so I added the 60...  now the reason why i added 45 to it was to be the 3rd choice. IRL the govern added 45acp to it buy using am adapter to the gun. This was also put in since they can't really find 60 easily which would just leave them with the shotgun shell. The 45 is to add single projectile range

Reply:
Fair, fair.
Right, come to think of it, if a dude shows up with this thing to kill me, I'd rather like him to have the gun loaded with magnum cartridges than shotgun shells.